letsdrill.rb

```diff
@@ -1,12 +1,27 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
+  if integer > 89
```

Comment:
On that 90-100 scale eh? 😆 Nice job, in Ruby, you don't need explicit return values if the thing you are intending to return is the last line in the method/scope.

Other things to thing about when you notice large conditionals are `case` statements, maybe try to update with one and see how those work and look.